core/domain/email_manager_test.py

```diff
@@ -1326,3 +1326,88 @@ def test_that_feedback_message_emails_are_correct(self):
             self.assertEqual(
                 sent_email_model.intent,
                 feconf.EMAIL_INTENT_FEEDBACK_MESSAGE_NOTIFICATION)
+
+
+class FlagExplorationEmailTest(test_utils.GenericTestBase):
+    """def setUp(self):
+        super(FlagExplorationEmailTest, self).setUp()
+
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+        self.signup(self.MODERATOR_EMAIL, self.MODERATOR_USERNAME)
+        self.moderator_id = self.get_user_id_from_email(self.MODERATOR_EMAIL)
+        self.set_moderators([self.MODERATOR_USERNAME])
+
+        self.exploration = self.save_new_default_exploration(
+            'A', self.editor_id, 'Title')
+        self.owner_ids = [self.editor_id]
+
+        self.report_text = 'AD'
+
+        self.can_send_emails_ctx = self.swap(
+            feconf, 'CAN_SEND_EMAILS', True)
+
+    def test_that_flag_exploration_emails_are_correct(self):
+        expected_email_subject = 'New flag has been raised for "Title"'
+
+        expected_email_html_body = (
+            'Hello Moderator,<br>'
+            'newuser has submitted a new report on the exploration Title'
+            'of the grounds of AD .'
+            '<a href="https://www.oppia.org/explore/A">"Title"</a>.<br>'
+            'The author/s of the exploration is/are editor, '
+            'You can modify the exploration by clicking '
+            '<a href="https://www.oppia.org/create/A">'
+            '"Edit Title"</a>.<br>'
+            '<br>'
+            'Thanks!<br>'
+            '- The Oppia Team<br>'
+            '<br>'
+            'You can change your email preferences via the '
+            '<a href="https://www.example.com">Preferences</a> page.')
+
+        expected_email_text_body = (
+            'Hello Moderator,\n'
+            'newuser has submitted a new report on the exploration Title'
+            'of the type AD, "Title"\n'
+            'The author/s of the exploration is/are editor, '
+            'You can modify the exploration by clicking "Edit Title"\n'
+            '\n'
+            'Thanks!\n'
+            '- The Oppia Team\n'
+            '\n'
+            'You can change your email preferences via the Preferences page.')
+
+        with self.can_send_emails_ctx:
+            email_manager.send_flag_exploration_email(
```

Comment:
Seems like you didn't modify the [database schema](https://github.com/oppia/oppia/blob/a2d844c6358fea285d7c9b92495bfa229ac16c65/core/storage/email/gae_models.py#L50) accordingly. Good thing you added tests!